frontend/src/routes/settings/connection/DatabaseConnectionForm.tsx

```diff
@@ -321,21 +308,23 @@ const TestingConnectionFragment = (props: {
       if (testConnectionResponse?.accessibleDatabases) {
         const baseDisplayName = data.displayName;
         for (const db of testConnectionResponse.accessibleDatabases) {
-          data.displayName = `${baseDisplayName} - ${db}`;
-          data.id = convertToAlphanumericDash(`${data.id}-${db}`);
```

Comment:
This would append the database for each new entry in accessibleDatabases to the ID. Instead, I'm no longer mutating the existing ID but just create a new object every time we invoke `createConnection`

Reply:
ah well that's a bit embarrassing 😅 Thank you!
Maybe I should have more frontend tests.